apps/emqx_modules/src/emqx_delayed.erl

```diff
@@ -143,7 +157,7 @@ on_message_publish(
             Timestamp ->
                 %% Check malicious timestamp?
                 Internal = Timestamp - erlang:round(Ts / 1000),
-                case Internal > ?MAX_INTERVAL of
+                case abs(Internal) > ?MAX_INTERVAL of
```

Comment:
```suggestion
                case Interval =< 0 orelse Interval > ?MAX_INTERVAL of
```
?

Reply:
we should maybe allow some time delay cause the timestamp to be lightly in the past.